app/repositories/AssessorRepository.scala

```diff
@@ -63,9 +63,35 @@ class AssessorMongoRepository(implicit mongo: () => DB)
     collection.update(query, saveBson, upsert = insertIfNoRecordFound) map assessorValidator
   }
 
-  override def countSubmittedAvailability: Future[Int] = {
-    val query = Json.obj(Seq("status" -> Json.toJsFieldJsValueWrapper(AssessorStatus.AVAILABILITIES_SUBMITTED.toString)): _*)
-    collection.count(Some(query))
+  def findAvailabilitiesForLocationAndDate(location: Location, date: LocalDate, skills: Seq[SkillType]): Future[Seq[Assessor]] = {
+    val query = BSONDocument("$and" -> BSONArray(
+      BSONDocument("skills" -> BSONDocument("$in" -> skills)),
+      BSONDocument("availability" ->
+        BSONDocument("$elemMatch" -> BSONArray(
+          BSONDocument("location" -> location),
+          BSONDocument("date" -> date),
+          BSONDocument("allocation" -> BSONDocument("$exists" -> false))
+        )))
+    ))
+
+    collection.find(query).cursor[Assessor]().collect[Seq]()
+  }
+
+  def assessorsForEvent(eventId: String): Future[Seq[Assessor]] = {
+    val query = BSONDocument("allocation" -> BSONDocument("$elemMatch" -> BSONDocument(
+      "id" -> eventId
+    )))
+
+    collection.find(query).cursor[Assessor]().collect[Seq]()
   }
 
+  // TODO Fix this when availability submission is complete
+  def countSubmittedAvailability: Future[Int] = Future.successful(0)
```

Comment:
Why it doesn't work? 
Availability submission is complete (FSET-1615). It's not assessor allocation, isn't it.